### PR TITLE
Inflatable QoL tweaks

### PR DESCRIPTION
**Pull request by Nerezza**
Features:
- Ctrl-click can deflate inflatables now.
- Using inflatables on adjacent tiles deploys them to that tile. Currently this doesn't care about objects in the way, ie computers or welding tanks, but it does respect glass and imo it works fine this way.

Bugfix:
- Bumping an inflatable door no longer freezes it closed for a few moments.
----
inflatables.dm is a mess so I'll probably look into cleaning it up to be more like Bay's sometime after Thanksgiving.

I'm also probably going to port the robot inflatables dispenser if Yosh doesn't do that, so engiborgs and crisis borgs can deploy airlocks where needed. However, I don't think the engineering derpartment should get dispensers like in Bay, hence the turf-deploy in this PR.

Edit: Oh and, this was tested.

### CODE REVIEW

**Comment by Anewbe** [comment]
Dispensers, you say?

**Comment by Nerezza** [comment]
The mechanical psuedo-cases in Bay's engineering that lets you turf-deploy and instantly deflate-collect inflatables (I'm sure you already know, but future reference.) Honestly I think that's fine for robots, but I'd prefer the cases we have now plus the turf deploy I put in above.

It just feels like we're going too far with equipping engineers with robotic dispensers, and aside from that I'm a little bothered with how said dispensers wipe out the fingerprints on their inflatables.

In other words, I'm being nitpicky but at least I put together an alternative that'd make engineer-portable dispensers redundant ;P
